rasa/core/training/interactive.py

```diff
@@ -1589,7 +1599,7 @@ def run_interactive_learning(
             "before_server_start",
         )
 
-    _serve_application(app, stories, skip_visualization)
+    _serve_application(app, stories, skip_visualization, server_args)
```

Comment:
let's pass in the `port` instead of `server_args` so we don't have to repeat the unpacking.

Reply:
ok. Though it will be useful in future to any new argument. Anyways will pass port directly and do the necessary changes.

Reply:
> Though it will be useful in future to any new argument.

What do you mean?